frontend/src/app/modules/posts/components/delete-post/delete-post.component.html

```diff
@@ -0,0 +1,21 @@
+<div *ngIf="visible" class="tooltip tooltip-bottom -top-4" data-tip="Delete">
+<span class="material-symbols-outlined relative -bottom-2" (click)="openModal('modal_'+postId)">
```

Comment:
może klasa cursor-pointer?

Reply:
hmm w tym <span>?

Reply:
na śmietniku. nie wiem jak ty masz ale u mnie bez tego dziwnie to wygląda bo mam cursor do zaznaczania tekstu jakby

Reply:
faktycznie, u mnie tak samo, zmienie to